src/azure-cli-core/setup.py

```diff
@@ -47,7 +47,7 @@
     'argcomplete~=1.8',
     'azure-cli-telemetry==1.0.6.*',
     'azure-mgmt-core>=1.2.0,<1.3.0',     # the preview version of azure-mgmt-core is 1.3.0b1, it cannot fit azure-core >=1.14.0
-    'cryptography>=3.2,<3.4',
+    'cryptography~=3.0',
```

Comment:
Using [Compatible release](https://www.python.org/dev/peps/pep-0440/#compatible-release) so that `cryptograph` is `<4`.